File: app/src/main/java/com/chandilsachin/diettracker/Food.java
package com.chandilsachin.diettracker;

/**
 * Created by Sachin Chandil on 29/04/2017.
 */

public class Food {
    public int foodId;
    public String foodName;
    public String foodDesc;
    public double protein;
    public double carbs;
    public double fat;
    public double calories;

    public Food(String foodName, String foodDesc, double protein, double carbs, double fat) {
        this.foodName = foodName;
        this.foodDesc = foodDesc;
        this.protein = protein;
        this.carbs = carbs;
        this.fat = fat;
        //this.calories = calories;
    }
}

File: app/src/main/java/com/chandilsachin/diettracker/Food.kt
package com.chandilsachin.diettracker

import android.arch.persistence.room.ColumnInfo
import android.arch.persistence.room.Entity
import android.arch.persistence.room.PrimaryKey

/**
 * Created by sachin on 22/5/17.
 */
class Food( var foodName: String,
           var foodDesc: String,
           var protein: Double,
           var carbs: Double,
           var fat: Double)
{
    var id: Long = 0
    var calories: Double = 0.toDouble()
}
